ChangeLog.md

```diff
@@ -2,21 +2,35 @@
 
 ## 2.0
 
-The main focus of this release was modularizing Happy.
+There are two main breaking changes in this release:
 
-* Generated parsers now activate the language extension `-XNoStrictData` without
-  which every use of a happy parser would lead to an immediate crash (#273).
-  This causes us to drop support for GHC < 8.0.
+1. Removal non-array, non-GHC modes, so flags `-ag` are the default now and
```

Comment:
"Removal of" or "Remove"

Reply:
Thanks, fixed (`Remove`)!

Reply:
So the candidates still have the typo, weirdly...

Reply:
Perhaps that was a caching issue. Regardless, in https://hackage.haskell.org/package/happy-lib-2.0/changelog it does say `Removed`, not `Removal`, for me.